src/main/java/pl/simpleascoding/tutoringplatform/dto/UpdateReviewDTO.java

```diff
@@ -0,0 +1,8 @@
+package pl.simpleascoding.tutoringplatform.dto;
+
+import org.hibernate.annotations.Immutable;
+import org.hibernate.validator.constraints.Range;
+
+@Immutable
+public record UpdateReviewDTO(String content, @Range(min = 1, max = 5) byte stars) {
```

Comment:
I don't think annotation @Range should be used on DTO

Reply:
Left the DTO validation intact, because of the following discussions:
https://stackoverflow.com/questions/20091826/should-dto-and-entity-both-have-input-validations
https://softwareengineering.stackexchange.com/questions/387763/should-i-validate-dtos-or-entities